terraform/fec_rds.tf

```diff
@@ -326,6 +326,29 @@ resource "aws_db_instance" "rds_staging_replica_1" {
   apply_immediately = true
 }
 
+resource "aws_db_instance" "rds_staging_replica_2" {
+  lifecycle {
+    prevent_destroy = true
```

Comment:
Next time for testing/temporary replicas, we should do `prevent_destroy = false`